lib/game.rb

```diff
@@ -25,45 +25,48 @@ def initialize(game_io: ConsoleIO.new, board_size:, player_2:, board_presenter:
 
     def play
         while status == :play
-            take_turn
+            output_board
+            input = get_valid_move(board: game_state.board, presenter: game_io)
```

Comment:
My nitpicky comment is that `get_valid_move` ultimately returns the valid position that the player wants to mark, right? So I think it would be clearer if you named the result of this function `position` and the passed that to `update_board`. Similarly, how could you change the name of the keyword parameter to `update_board` so that it is more clear about the value that it's actually using to update the board? On the other hand, I see that `update_board` actually expects a string (because when you get input from the command line, it comes back as a string) and converts it to an int to mark the board, so maybe the name input makes sense... If you want to keep it as input, I think it's a bit vague (just something to think about)

Reply:
So, this method does this:

1. Takes a game state and IO
2. Returns a string from the user input?

Reply:
Yup, takes a string from input/output, then passes it to update board.
To make it clearer in the play flow, I'll rename `input` to `position`